### PR TITLE
[0.7.0]: Add versioning-0.7.0

### PR DESCRIPTION
**Pull request by anupriya0703**
- [x] Adding of version 0.7.0

Screenshot from 2019-10-14 19-16-11

Signed-off-by: anupriya0703 <anupriya.gupta@mayadata.io>

### CODE REVIEW

**Review by qiell — CHANGES_REQUESTED** [review]
Need to add a sidebar file for 0.7 in `versioned_sidebars` directory

File: website/siteConfig.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ const users = [
 ];
 
 const siteConfig = {
-  title: 'Litmus Docs', // Title for your website.
+  //title: 'Litmus Docs', // Title for your website.
```
Comment (qiell) [inline website/siteConfig.js:24]: Why it is commented?

**Review by qiell — COMMENTED** [review]
Sidebar file has several files but I can only see one file in version 7 docs. Can you check it?

File: website/siteConfig.js (hunk 1 above)
Comment (anupriya0703) [reply website/siteConfig.js:24]: I'm  raising a new PR for update the logo, which already has the title.

**Comment by anupriya0703** [comment]
> Sidebar file has several files but I can only see one file in version 7 docs. Can you check it?

Can you please point the file, unable to figure out the file

File: website/siteConfig.js (hunk 1 above)
Comment (qiell) [reply website/siteConfig.js:24]: Ok.

**Review by qiell — DISMISSED** [review]
lgtm

**Review by qiell — APPROVED** [review]
lgtm

**Review by chandankumar4 — APPROVED** [review]
lgtm

**Pull request merged**